index.md

```diff
@@ -30,9 +30,9 @@ description: Vanilla OS is an Immutable Linux-based distribution which aims to p
 <div class="hero hero-center">
     <div class="container hero-wrapper">
         <div class="hero-text">
-            <h2>Nicely Updated.</h2>
+            <h2>Up to Date.</h2>
             <p>Designed to last over time and always be faithful to you.</p>
-            <p>Vanilla OS is an <span class="mark">immutable</span> and <span class="mark">atomic</span> Ubuntu Linux-based Point Release distribution, that receives updates at the right time, neither before nor after, without sacrificing security and functionality.</p>
+            <p>Vanilla OS is an <span class="mark">immutable</span> and <span class="mark">atomic</span> Ubuntu-based Point Release distribution that receives updates at the right time without sacrificing security or functionality.</p>
```

Comment:
```suggestion
            <p>Vanilla OS is an <span class="mark">immutable</span> and <span class="mark">atomic</span> Ubuntu-based Point Release distribution that receives updates at the right time without sacrificing security or functionality.</p>
```

Personally don't think "Point Release" is needed here, since Ubuntu already makes it a point release.

Reply:
Fair enough